pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[project]
+name = "signalilo-scrubbed"
+description = "Signalilo with alert scrubber included"
+version = "0.0.0"
+authors = [
+    { name = "Adfinis", email = "support@adfinis.com" }
+]
+
+dependencies = [
+    "Flask==3.0.3",
+    "requests==2.32.3",
+    "pyinstaller==6.8.0",
+    "waitress==3.0.0",
+]
+
+[project.optional-dependencies]
+test = [
+    "ruff==0.5.0",
+    "types-waitress==3.0.0.20240423",
+    "types-requests==2.32.0.20240622",
+    "pytest==8.2.2",
+    "pytest-cov==5.0.0",
+    "requests-mock==1.12.1",
+]
+
+[tool.flit.module]
+name = "scrubbed"
+
+[build-system]
+build-backend = "flit_core.buildapi"
```

Comment:
I usually lean towards using [`poetry`](https://python-poetry.org/) for everything so align myself with other efforts in the enterprise. [`flit`](https://flit.pypa.io/) is much more lightweight in comparison, and it doesn't really matter during runtime anyway.

I did not know about/remember [`pyinstaller`](https://pyinstaller.org/) and the only reason i touched these parts is because it helped me get pytest up and running without more complex folders whatnot.

For sure something we want to revisit before merging this, I'll gladly refactor back to requirement*txt if we want to stick with that route.

Reply:
going with pyproject lgtm too